packages/ember-htmlbars/tests/helpers/hash_test.js

```diff
@@ -7,67 +7,65 @@ import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
 
 var view;
 
-if (isEnabled('ember-contextual-components')) {
```

Comment:
This was introduced in 2.3.0. The rest of changes in this file are just formatting.